src/auth/models.cpp

```diff
@@ -30,6 +32,10 @@ DEFINE_string(auth_password_strength_regex, default_password_regex.data(),
 
 namespace memgraph::auth {
 namespace {
+const std::string ASTERISK = "*";
+const uint64_t LabelPermissionMax = static_cast<uint64_t>(memgraph::auth::LabelPermission::CREATE_DELETE);
```

Comment:
It's highest value(uint64_t-wise) in memgraph::auth::LabelPermission enum. Since i needed to mask against the highest value it seemed logical to introduce this constant